script.js

```diff
@@ -66,7 +66,7 @@ document.addEventListener("DOMContentLoaded", () => {
       thNames.forEach((thName) => {
         const td = createEl({
           el: "td",
-          text: item[thName],
+          text: createUniqTd(thName, item.body, unique, item),
```

Comment:
If you use more than 2-3 params- use data transfer object (dto)
https://stackoverflow.com/questions/1051182/what-is-a-data-transfer-object-dto